### PR TITLE
Feature/gradient clip

### PR DESCRIPTION
**Pull request by MathGaron**
Might be useful, especially for those of you who use LSTMs...

### CODE REVIEW

**Comment by jacenfox** [comment]
I don't get why the max_norm is set to 1.
[L139](https://github.com/MathGaron/pytorch_toolbox/blob/600a7b300435999bb1bd118e8311b8d063b719d0/pytorch_toolbox/train_loop.py#L139 )               torch.nn.utils.clip_grad_norm(self.model.parameters(), 1)

**Comment by MathGaron** [comment]
People usually use 1, (if it gets higher than that you have the exploding gradient problem!) mmh maybe for some unusual case people could need to clip it lower than 1.. but for now I guess that it should be fine?

**Comment by MathGaron** [comment]
Bump!?
